ISISNeutronMuon/MDANSE
# Atom mapping update

**Description of work**
- Updated the atom mapping in DL_POLY and added the ability to map atoms for all other converter except the ones which use a PDB file input.
- Added a atom mapping widget to assist with atom mapping input creation.
- Fixed CASTEP trajectory conversion.
- Removed LAMMPS mass tolerance and smart mass association since atoms can be mapped manually.
- Improved atom mapping guess so that mass information is used when supplied. Zero mass atoms are set to dummy atoms.
- Updated 3D view table so that it shows the isotope that is used.

Atom mapping not added for converters which used PDB file input since the mapping is done with the molecule, nucleotide and residue databases.

![Animation10](https://github.com/ISISNeutronMuon/MDANSE/assets/14276033/b024cf93-7e56-428a-8f83-938c84cb3730)

**Fixes**
Closes #336, Closes #335, Closes #334, Closes #306, Closes #330

**To test**
- Open a conversion for any converter with atom mapping. 
- Using the helper widget try mapping the atoms to something else like Au.
- Convert the trajectory and load it up in the trajectory viewer.
- From the 3D viewer check that the atoms have change to the selected ones.
- Repeat for a few trajectory converters.
